src/main/java/uk/gov/hmcts/reform/cwrdapi/service/impl/CaseWorkerServiceImpl.java

```diff
@@ -411,6 +444,8 @@ public CaseWorkerProfile createCaseWorkerProfile(CaseWorkersProfileCreationReque
                         = (UserProfileCreationResponse) requireNonNull(responseEntity.getBody());
                     updateUserRolesInIdam(cwrdProfileRequest, userProfileCreationResponse.getIdamId());
                 }
+                //Avoid select Insert for new Inserts
+                setNewCaseWorkerProfileFlag(caseWorkerProfile);
```

Comment:
is it right place? as method name updateCaseWorkerProfiles not sure

Reply:
Had a discussion. Call is at right place